src/module_2/module_2.md

```diff
@@ -0,0 +1,3512 @@
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>id</th>
+      <th>user_id</th>
+      <th>created_at</th>
+      <th>order_date</th>
+      <th>user_order_seq</th>
+      <th>ordered_items</th>
+      <th>variant_id</th>
+      <th>price</th>
+      <th>compare_at_price</th>
+      <th>vendor</th>
+      <th>product_type</th>
+      <th>tags</th>
+      <th>_merge</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>2204073066628</td>
+      <td>62e271062eb827e411bd73941178d29b022f5f2de9d37f...</td>
+      <td>2020-04-30 14:32:19</td>
+      <td>2020-04-30</td>
+      <td>1</td>
+      <td>33618849693828</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>left_only</td>
+    </tr>
+    <tr>
+      <th>107958</th>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaT</td>
+      <td>NaT</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>39788282282116</td>
+      <td>8</td>
+      <td>8</td>
+      <td>betteryou</td>
+      <td>superfoods-supplements</td>
+      <td>[cruelty-free, gluten-free, vitamins-supplements]</td>
+      <td>right_only</td>
+    </tr>
+    <tr>
+      <th>107959</th>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaT</td>
+      <td>NaT</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>39349425242244</td>
+      <td>6</td>
+      <td>7</td>
+      <td>nivea</td>
+      <td>suncare</td>
+      <td>[suncream]</td>
+      <td>right_only</td>
+    </tr>
+    <tr>
+      <th>107960</th>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaT</td>
+      <td>NaT</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>40155145076868</td>
+      <td>2</td>
+      <td>2</td>
+      <td>fairy</td>
+      <td>dishwashing</td>
+      <td>[washing-up-liquid]</td>
+      <td>right_only</td>
+    </tr>
+    <tr>
+      <th>107961</th>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>NaT</td>
+      <td>NaT</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>40070658490500</td>
+      <td>0</td>
+      <td>0</td>
+      <td>jordans</td>
+      <td>cereal</td>
+      <td>[cereal-bars]</td>
+      <td>right_only</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+
+```python
+# We do a quick manual check. This should not exist in orders dataframe
+if not (df_orders_unrolled['ordered_items'] == '39788282282116').any():
+    print("Not exist indeed")
+```
+
+    Not exist indeed
+
+
+This means there are 256 unique ids in the inventory not being sold, and 1 sale with no correspondence in inventory. I would send this previous dataframe to Sales team in order to confirm it, but in this case we would observe that those might be de-listed EANs (past items in the inventory no longer available)
```

Comment:
Why are you concerned about an entire _sale with no correspondence in inventory_, instead of individual items?